scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -569,11 +569,14 @@ def handle(self):
                 try:
                     (stdout, stderr) = proc.communicate(timeout=60)
                     # ignore unencodable unicode chars
-                    logger.debug(stdout.decode(errors = "ignore"))
+                    decoded_stdout = stdout.decode(errors="ignore")
+                    decoded_stderr = stderr.decode(errors="ignore")
+                    logger.debug("Flash command stdout: %s", decoded_stdout)
+                    logger.error("Flash command stderr: %s", decoded_stderr)
```

Comment:
stderr is already going to be written to `d_log` - see few lines below, isn't it ?

Reply:
It doesn't make much sense to only output errors to a file. When i turn on twisters most verbose logging setting i should get errors on stdout. Even though this outputs to a file, its hard to find where the error begins and where it ends.